### ISSUE TITLE
use astroquery instead of SDSS class ? 

### ISSUE THREAD

**Issue by ismael-mendoza** [prob-ml/bliss#441]
We talked about this in the last meeting, it might be worth doing some refactoring if astroquery has a nicer/more standard interface

https://astroquery.readthedocs.io/en/latest/sdss/sdss.html

**Comment by dereklhansen** [comment]
I agree this might be a nice change, but I think we should wait to implement it when we need to pull more data in that what we have now. The code in https://github.com/prob-ml/bliss/blob/957d5ce840eba1af0120d2ed04f3a416dd151843/bliss/datasets/sdss.py is quite simple and does what we need it to.

**Comment by jeff-regier** [comment]
I'm not sure whether astroquery gives us all the SDSS data we need. I see it downloads the frame files, but we also need the files describing the PSF and some calibration values (e.g. gain). These values are stored in the "photoField" files:

https://github.com/prob-ml/bliss/blob/72afe9f76e10edc66cad3323adfca8c13dd4723a/bliss/datasets/sdss.py#L36

**Comment by jeff-regier** [comment]
On closer inspection, it appears the `gain` is the only attribute we're using from the "photoField" files. Everything else is in the "frame" files, which astroquery.SDSS downloads. I'm not sure whether downloading gain issue essential. It may be that the gain is always around 4.

**Comment by jeff-regier** [comment]
Another issue: It's not clear how testing with SDSS data will work if we use astroquery because astroquery fetches data from a remote server (rather than git-lfs). The SDSS servers are somewhat flaky, so we wouldn't want to rely on them for our tests to run.

Using astroquery may be more trouble than it's worth. The existing code works and it isn't particularly complicated. Ultimately we want to move away from SDSS (and on to LSST, Euclid, and DeCALs) anyway.

**Comment by ismael-mendoza** [comment]
That makes sense, thanks Jeff for looking into this
